addons/stock/models/stock_lot.py

```diff
@@ -147,6 +152,27 @@ def _compute_delivery_ids(self):
             lot.delivery_ids = delivery_ids_by_lot[lot.id]
             lot.delivery_count = len(lot.delivery_ids)
 
+    def _compute_partner_ids(self):
+        delivery_ids_by_lot = self._find_delivery_ids_by_lot()
+        for lot in self:
+            if len(delivery_ids_by_lot[lot.id]) > 0:
```

Comment:
``` suggestion
            if delivery_ids_by_lot[lot.id]:
```
No need to check len, non-empty lists evaluate as True, and empty lists evaluate as False

Reply:
Done.